image_topic: '/usb_cam/image_raw'
camera_matrix:
  rows: 3
  cols: 3
  data: [ 479.5879067840989, 0.0, 310.263858232875, 0.0, 479.7873512067808, 186.01960806843329, 0.0, 0.0, 1.0 ]
distortion_coefficients:
  rows: 1
  cols: 5
  data: [ 0.06434145176179719, -0.24430016101408403, -0.005081429565207777, -0.001826222404770314, 0.2541648131826896 ]
